File: Frontend/trackr-administrador/src/app/shared/dtos/gestion-expediente/expediente- recomendacion-form-dto.ts
export class ExpedienteRecomendacionFormDTO
{
    public idExpedienteRecomendacion: number;
    public idExpediente: number;
    public fecha: Date ;
    public descripcion: string;
    public idDoctor: number;

    constructor() {
        this.fecha = new Date();
    }
}

File: Backend/TrackrAPI/TrackrAPI/Dtos/GestionExpediente/ExpedienteRecomendacionFormDTO.cs
namespace TrackrAPI.Dtos.GestionExpediente;

public class ExpedienteRecomendacionFormDTO
{

    public int IdExpedienteRecomendacion { get; set; }
    public string? Descripcion { get; set; }
    public int IdUsuario { get; set; }
    public int IdDoctor { get; set; }
    public DateTime Fecha { get; set; }

}
